### PR TITLE
Feature/state california

### PR DESCRIPTION
**Pull request by Tommi-Tsuruga**

### CODE REVIEW

**Comment by zzolo** [comment]
You will need to add the `psutil` dependency to `requirements.txt` and `setup.py`.  This should fix the tests.

File: reggie/configs/data/california.yaml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+  - CountyCode
+  - RegistrantID
+  - ExtractDate
+  - ElectionType
+  - ElectionTypeDesc
+  - ElectionDate
+  - ElectionName
+  - Method
+county_names:
+  Alameda: 1
```
Comment (cvcotton) [inline reggie/configs/data/california.yaml:210]: for readability, just make sure to alphabetize these county names

File: reggie/configs/primary_locale_names/county/california.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+[
+  { "name": "Alameda", "id": "1" },
+  { "name": "Orange", "id": "30" },
```
Comment (cvcotton) [inline reggie/configs/primary_locale_names/county/california.json:3]: and also alphabetize these here

File: reggie/configs/primary_locale_names/county/california.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  { "name": "Alpine", "id": "2" },
+  { "name": "Placer", "id": "31" },
+  { "name": "Amador", "id": "03" },
```
Comment (cvcotton) [inline reggie/configs/primary_locale_names/county/california.json:6]: MAKE SURE to remove the leading zero from "03", or the notification emails may be messed up I think

**Review by cvcotton — CHANGES_REQUESTED** [review]
Fix up the county names as I commented, and then this looks ready to merge

**Review by cvcotton — APPROVED** [review]
cool, thanks!

**Pull request merged**